src/evaluations/data/evaluation_configs.py

```diff
@@ -44,6 +44,7 @@
     MAX_FREQUENCY)
 
 NUM_RUNS_VALUE = 100
+SMOKE_TEST_UNIVERSE_SIZE = 200_000
 UNIVERSE_SIZE_VALUE = 1000000
```

Comment:
can you change this file to consistently use underscores for all large numbers?  Not that big a deal but when I went to review it stuck out

Reply:
Done.